API.md

```diff
@@ -0,0 +1,65 @@
+# Octoliner API
+
+## `class Octoliner`
+
+Create an object `Octoliner` to communicate with a particular Octoliner board.
+
+### `Octoliner(i2c_address=42)`
+
+The constructor for Octoliner class. The `i2c_address` specifies the board
+I²C address which is `42` factory-default but can be changed programmatically.
+
+### `set_sensitivity(sense: float) -> None`
+
+Sets the sensitivity of the photodetectors in the range from `0` to `1.0`.
+
+### `analog_read(sensor: int) -> float`
+
+Reads the value from one line sensor. Return value in range from `0` to `1.0`.
+
+### `analog_read_all(analog_values: list) -> None`
+
+Reads all 8 channels to the `analog_values` list.
```

Comment:
Not pythonic. Should be:

`analog_read_all() -> list(float)`

Reply:
Should a function return a new list, and not modify an existing one?